fvm/fvm_bench_test.go

```diff
@@ -212,23 +212,39 @@ func (b *BasicBlockExecutor) SetupAccounts(tb testing.TB, privateKeys []flow.Acc
 	accounts := make([]TestBenchAccount, 0)
 	serviceAddress := b.Chain(tb).ServiceAddress()
 
-	accontCreationScript := `
```

Comment:
Maybe we can use the fixtures above instead?

Reply:
fixed in https://github.com/onflow/flow-go/pull/2344